src/app/modules/user-teacher/teacher-post/components/teacher-create-post-form/teacher-create-post-form.component.ts

```diff
@@ -11,31 +11,37 @@ import { TeacherAddPost } from '../../store/actions';
 export class TeacherCreatePostFormComponent implements OnInit {
 
   createPostForm!: FormGroup;
-  
+
   constructor(
     private fb: FormBuilder,
```

Comment:
Предложение по названию переменной. Мне кажется, такие короткие переменные в будущем могут создать неприятности, если их придется искать в большом количестве кода. Будет находить всякий мусор типа такого:

![image](https://user-images.githubusercontent.com/71533482/109888646-81360980-7c8c-11eb-93fd-d1595a2efd25.png)

Я бы написал более уникальный, хотя бы так:
```suggestion
    private formBuilder: FormBuilder,
```